### PR TITLE
fix: fix action button theme

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[20230512 transparent 的 mobile sidemenu 的按鈕看不到（因為是 transparent theme，但應該用 normal 就好）](https://app.asana.com/0/1203983432530614/1204588802643715/f)

# Notice

# Dependency

### CODE REVIEW

File: packages/universal-header/src/components/action-button-item.js [hunk 1]
```diff
@@ -76,7 +76,8 @@ const ActionButtonItem = ({
 }) => {
   const { theme } = useContext(HeaderContext)
   const buttonTheme =
-    direction === DIRECTION_TYPE.row || theme === THEME.photography
+    (direction === DIRECTION_TYPE.row && theme !== THEME.transparent) ||
```
Comment (Aylie-Chou) [inline packages/universal-header/src/components/action-button-item.js:79]: The `action button` on header would need to use `transparent ` theme may has direction type `row` & theme `transparent`.
螢幕截圖 2023-06-19 下午1 18 41

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
Current code would also make header action button to use `normal` theme. (which would need to use transparent theme component)

File: packages/universal-header/src/components/action-button-item.js (hunk 1 above)
Comment (liruchen32) [reply packages/universal-header/src/components/action-button-item.js]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**